csharp/src/Drivers/BigQuery/BigQueryStatement.cs

```diff
@@ -75,6 +76,24 @@ public override QueryResult ExecuteQuery()
             readClientBuilder.Credential = this.credential;
             BigQueryReadClient readClient = readClientBuilder.Build();
 
+            if (results.TableReference == null)
+            {
+                ListJobsOptions listJobsOptions = new ListJobsOptions();
```

Comment:
This doesn't look like obvious logic. If there's specific documentation for BigQuery which describes this behavior then it would be good to link to that from the source code.

Reply:
Good advice, here is the relevant official documentation: https://cloud.google.com/bigquery/docs/multi-statement-queries#get_all_executed_statements

Reply:
Am I waiting for a new iteration? Were you going to add the link in the source code?

Reply:
May I ask if this means adding the link in the source code as a comment? If so, I have already submitted a new commit with the link added. Kindly review it at your convenience. Thank you for your support.

Reply:
Yes, thank you! A checkin hook is complaining about mixed line endings here. Can you please fix?

Reply:
fixed, thank you~